src/UI/Buyer/src/app/shared/components/line-item-card/line-item-card.component.html

```diff
@@ -4,7 +4,7 @@
       <div class="col-3 p-0">
         <img class="img-thumbnail"
              appFallbackImage
-             [src]="lineitem.Product.xp?.primaryImageURL || 'http://placehold.it/300x300'">
+             [src]="lineitem.Product.xp?.imageURLs[0] || 'http://placehold.it/300x300'">
```

Comment:
We'll want a safe navigation operator (`?.`) for Product.xp as well since xp can be null.